metrics/metrics.yaml

```
@@ -186,12 +162,22 @@
   attributes:
   - attribute-name: read_type
     attribute-type: string
-    values: *read_types_list
+    values:
+    - "Parallel"
+    - "Random"
+    - "Sequential"
+    - "buffered"
```

Let's follow the convention and keep it titlecased.
```suggestion
    - "Buffered"
```